app/views/content/train-to-be-a-teacher/promos/_find-your-course.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="promo-content find-your-course">
+  <div class="left">
+    <div class="icon">
+       <!-- we don't have a black magnifying glass icon yet -->
```

Comment:
@peteryates Is a black magnifying glass icon something that @gazcarr can help with ?